### DISCUSSION TITLE
Script "Jumped" not working

### DISCUSSION THREAD

**Discussion by Galushkin23** [EDCD/EDDI#2433] [Troubleshoot Scripts] [answered]
When I press "test" button it responded: "Error with jump script. Object reference ... something ..." It's in Elite Odyssey, btw.

**Comment by Tkael** [comment, marked as answer]
That sounds like the same problem some others are reporting in issue #2431 . I haven't been able to replicate the problem myself yet though.

**Reply by Tkael** [reply]
Please report any information you can provide within that ticket.
